lightningd/peer_htlcs.c

```diff
@@ -1256,6 +1282,14 @@ static struct channel_id *calc_forwarding_channel(struct lightningd *ld,
 					 channel_scid_or_local_alias(best)));
 	}
 
+	log_info(hp->channel->log,
+		 "Decided to forward htlc_id=%" PRIu64
+		 " over channel with scid=%s with peer %s",
+		 hp->hin->dbid,
```

Comment:
hp->hin->dbid isn't the right thing here: it's meaningless unless you have db access?  You want  hp->hin.key.id.

Above, too...